web/src/beta/features/WorkspaceSetting/innerPages/Members/Members.tsx

```
@@ -0,0 +1,472 @@
+                  </UserInfo>
+                  <DeleteIcon
+                    icon="trash"
+                    size="normal"
+                    hasBorder={false}
+                    appearance="simple"
+                    onClick={() => {
+                      handleDeleteUserForSearchResult(memberSearchResult);
+                    }}
+                  />
+                </ItemContainer>
+              ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)